DocumentsFromSnapshotMigration/docker/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+# Fail the script if any command fails
+set -e
+
+# Print our ENV variables
+echo "RFS_COMMAND: $RFS_COMMAND"
```

Comment:
Looks like we support `--target-password` here so let's make sure we don't log it.

Reply:
Ugh, good catch.